### DISCUSSION TITLE
Result list sorting

### DISCUSSION THREAD

**Discussion by kaiurin** [TarekRaafat/autoComplete.js#404] [Q&A]
Now searched list looks chaotic, there is no order way from the most relative result to request to less. For example between string that begins from pasta here is "soup" string.

![image](https://user-images.githubusercontent.com/43986607/226459131-0d620794-997c-4883-aee9-6c1163393c7d.png)

Is there any way to sort result in this way?

**Comment by gagecarto** [comment]
I am having the same issue.. I saw this fix but it makes no change

```
filter: (list) => {            
            const results = list.filter((item) => {
                const inputValue = autoCompleteJS.input.value.toLowerCase();
                const itemValue = item.value.toLowerCase();

                if (itemValue.startsWith(inputValue)) {
                    return item.value;
                }
            });
            return results;
        }

```
